### ISSUE TITLE
Request: Add examples of using this with build123d

### ISSUE THREAD

**Issue by KilowattSynthesis** [meadiode/cq_gears#20]
It would be awesome if there were examples of using parts generated with this in build123d, as there is no gear generator for build123d, as far as I can tell.

**Comment by erooke** [comment]
https://build123d.readthedocs.io/en/latest/tips.html#build123d-cadquery-integration

**Comment by KilowattSynthesis** [comment]
Yeah I understand; I still think a good, self-contained example in these docs would be helpful though! Learning CadQuery isn't really something I have time for at the moment, so it'd be nice to be able to use this right away with a straight-forward example of loading right into Build123d

**Comment by erooke** [comment]
For sure. I was just linking how this works in general to help out future travelers, yourself, or a contributor who wants to write said examples.

Shouldn't require learning any CadQuery; take any example code which is already present and then do:

```python
import build123d as b3d
b3d_solid = b3d.Solid.make_box(1,1,1)

####################
# existing example #
####################

b3d_solid.wrapped = cq_solid.wrapped
```
